backend/src/main/java/com/huellapositiva/domain/actions/ResendEmailConfirmationAction.java

```diff
@@ -38,7 +38,7 @@ public void execute(String accountId) {
         JpaCredential jpaCredential = jpaCredentialRepository.findByAccountId(accountId)
                 .orElseThrow(() -> new UsernameNotFoundException("User with account ID: " + accountId + " was not found."));
 
-        if (jpaCredential.getEmailConfirmed()) {
+        if (Boolean.TRUE.equals(jpaCredential.getEmailConfirmed())) {
```

Comment:
I suppose you made this to avoid using a Boolean object in the if. Would it not be better to use the .booleanValue() method of Boolean to get the primitive instead of an equals()?

Reply:
SonarLint rules!